packages/drivers/tinylicious-driver/package.json

```diff
@@ -40,9 +40,9 @@
 		"@fluidframework/core-interfaces": "workspace:~",
 		"@fluidframework/driver-definitions": "workspace:~",
 		"@fluidframework/driver-utils": "workspace:~",
-		"@fluidframework/protocol-definitions": "^1.1.0",
+		"@fluidframework/protocol-definitions": "^3.0.0",
 		"@fluidframework/routerlicious-driver": "workspace:~",
-		"@fluidframework/server-services-client": "^1.0.1",
+		"@fluidframework/server-services-client": "^2.0.0",
```

Comment:
The only use of this server dep seems to be the [`getRandomName()` function](https://github.com/microsoft/FluidFramework/blob/6cf27747da42420714c5eeadcb51d0a689fe3e3f/server/routerlicious/packages/services-client/src/generateNames.ts#L8) which I don't think deserves a dep on server packages. I'll create a task to bring that function somewhere else to remove this dep.